TTSWorkflow.go

```diff
@@ -45,7 +46,7 @@ func TTSWorkflow(ctx workflow.Context, fileInputPath string) (string, error) {
     if err != nil {
         return "", fmt.Errorf("failed to create temporary file: %w", err)
     }
-
+	// We should not use `log` in a workflow. users should call log := workflow.GetLogger(ctx) and use that logger.
```

Comment:
Workflow logger updated to use workflow.GetLogger(ctx). I'm going to discover if the format statements still work or I need to fix further.

Reply:
```
2024/08/22 12:50:14 INFO  Created temporary file for processing: %s Namespace default TaskQueue tts-task-queue WorkerID 66373@Horton.local@ WorkflowType TTSWorkflow WorkflowID my-test-1 RunID ef481314-5154-426e-b3fb-0a9950d1b1fc Attempt 1 /var/folders/fl/_pmjcnv520v5bsh9_d_kgn5c0000gn/T/2255158290.tmp
```

Updating.

Reply:
Flip-flopped message assignment and logger output. Logger just outputs the message now.